internal/ui/graph/default_row_iterator.go

```diff
@@ -132,10 +133,20 @@ func (s *DefaultRowIterator) Render(r io.Writer) {
 				style = style.Inherit(s.textStyle)
 			}
 
+			isAce := func() bool {
+				return s.AceJumpPrefix != nil &&
+					(segment.Text == row.Commit.ChangeId || segment.Text == row.Commit.CommitId) &&
```

Comment:
> I still have to fix a bug where the `ace` bookmark is somehow identified as a commit_id

PR is ready, but I guess I need help with this, because it seems  we have a bug where sometimes a bookmark is identified as a commit_id. I'm checking here that the segment text is either the row changeId or the commitId, but somehow `ace` (a bookmark -- see my screenshot) is sometimes (not always) identified a commit id and this condition evaluates to true, making ace jumping into the bookmark instead of the real commit id.

Reply:
Ah, yes that might happen. Let me have a look.